jena-arq/src/main/java/org/apache/jena/sparql/exec/http/QueryExecHTTP.java

```diff
@@ -119,10 +117,28 @@ public static QueryExecHTTPBuilder service(String serviceURL) {
     private HttpClient httpClient = HttpEnv.getDftHttpClient();
     private Map<String, String> httpHeaders;
 
+    public QueryExecHTTP(String serviceURL, Query query, String queryString, int urlLimit,
```

Comment:
`public` does not like it is necessary

So Jena6, shall we make it `protected` and have only one constructor.

If so, add `@Deprecated` with a javadoc comment saying the constructor will become protected on both constructors.

Reply:
Done. The new constructor no longer uses the now deprecated override header, whereas the original one still supports it.